src/main/java/taskitems/TaskManager.java

```diff
@@ -0,0 +1,236 @@
+package taskitems;
+
+import taskitems.exceptions.IllegalInputException;
+import taskitems.task.Deadline;
+import taskitems.task.Event;
+import taskitems.task.Task;
+import taskitems.task.Todo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class TaskManager {
+    private final String PATH = "data.txt";
+    private ArrayList<Task> tasks = new ArrayList<>();
+    private ArrayList<Task> bin = new ArrayList<>();
```

Comment:
The ArrayList type is usually for a collection of objects. You could use a name in [plural form](https://se-education.org/guides/conventions/java/basic.html#naming) here.